src/client/index.js

```diff
@@ -24,11 +26,30 @@ const targetNode = document.getElementById('root')
 // run a router match (not sure why this is necessary)
 match({ routes, history }, (error, redirectLocation, renderProps) =>
   render(
-    <Router
-      render={renderAsyncProps}
-      routes={routes}
-      {...renderProps}
-    />,
+    <AppContainer>
```

Comment:
`<AppContainer>` manages state in `development` mode and does nothing but render it's children in `production`